capstone/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -5,28 +5,32 @@
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
-import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.appengine.api.datastore.Entity;
 import java.util.List;
 import java.util.ArrayList;
 import com.google.gson.Gson;
 import com.google.sps.data.Form;
-import com.google.sps.data.RequestJsonParser;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.sps.service.DatabaseService;
+import com.google.sps.data.RequestParser;
 
 @WebServlet("/form-handler")
 public class FormHandlerServlet extends HttpServlet {
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        List<Form> forms = new ArrayList<>();
-        Form form = null;
         Query query = new Query ("Form");
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         PreparedQuery results = datastore.prepare(query);
+        
+        List<Form> forms = new ArrayList<>();
+        Form form = null;
```

Comment:
Inline

There is no need to declare a forward variable if it is only used once in the following line. So you can simplify your adding to just say 
forms.add(new Form(entity));